src/api/discord/download.js

```diff
@@ -41,7 +43,12 @@ async function download(title) {
             // reversing as the latest message will be featched first
             const reversedMessages = messages.reverse();
             reversedMessages.forEach(msg => {
-                console.log(`${msg.author.tag}: ${msg.content}`);
+                // console.log(`${msg.author.tag}: ${msg.content}`);
+                // console.log(msg.attachments);
+                if(msg.attachments.size > 0) {
+                    const fileUrl = msg.attachments.first().url;
+                    request(fileUrl).pipe(fs.createWriteStream(`./downloads/${title}.jpg`));
+                }
```

Comment:
Add a console message for user to understand that the download has been completed and can be located <location_name>.